pos_show_clock/static/src/js/ChromeWidgets/Clock.js

```diff
@@ -0,0 +1,40 @@
+odoo.define("pos_show_clock.Clock", function (require) {
+    "use strict";
+
+    const PosComponent = require("point_of_sale.PosComponent");
+    const Registries = require("point_of_sale.Registries");
+    const useState = owl;
+    // V16 const onMounted = owl;
+
+    class Clock extends PosComponent {
+        setup() {
+            super.setup();
+            this.state = useState({current_time: ""});
+            // V16 onMounted(this.onMounted);
+            this.mounted();
+        }
+        mounted() {
+            // V16
+            // onMounted() {
+            this.updateCurrentTime();
+        }
+        updateCurrentTime() {
+            var self = this;
+            const now = Date.now();
+            const next_update = 60050 - parseInt(moment(now).format("ss")) * 1000;
```

Comment:
Dont understand what is 60050 ? Could you add a comment ?

Reply:
- 1 minute has 60000ms
- 60000 + 50(less then 100ms rule), is the time we can update the minute without the user noticing the difference when comparing it to another watch.